app/views/admin/world_location_news/edit.html.erb

```diff
@@ -1,30 +1,67 @@
-<% page_title "Edit " + @world_location_news.name %>
+<% content_for :page_title, "Edit " + @world_location_news.name %>
+<% content_for :title, "Edit " + @world_location_news.name %>
+<% content_for :context, "World location" %>
+<% content_for :title_margin_bottom, 4 %>
+<div class="govuk-!-margin-bottom-8">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= form_for([:admin, @world_location_news]) do |form| %>
+        <%= form.fields_for :world_location, @world_location do |location_form| %>
+          <%= render "govuk_publishing_components/components/warning_text", {
+            text: "Changes to worldwide locations appear instantly on the live site.",
+          } %>
 
-<div class="row">
-  <section class="col-md-8">
-    <h1>Edit world location</h1>
-  <p class="warning">Warning: changes to worldwide locations appear instantly on the live site.</p>
-    <%= form_for([:admin, @world_location_news]) do |form| %>
-      <%= form.fields_for :world_location, @world_location do |location_form| %>
-        <div class="form-group">
-          <%= location_form.label :world_location_type, 'Type' %>
-          <%= location_form.select :world_location_type, WorldLocation.world_location_types.keys.map {|key| [I18n.t("world_location.type.#{key}", count: 1), key]}, {}, {class: 'form-control input-md-3'} %>
-        </div>
-
-        <%= form.text_field :title %>
+          <%= render "govuk_publishing_components/components/select", {
+            id: "world_location_news_world_location_attributes_world_location_type",
+            name: "world_location_news[world_location_attributes][world_location_type]",
+            label: "Type",
+            heading_size: "l",
+            options: WorldLocation.world_location_types.keys.map { |location_type| { text: I18n.t("world_location.type.#{location_type}", count: 1), value: location_type } }
+          } %>
 
-        <%= form.text_area :mission_statement, rows: 20, class: "previewable", data: {
-          module: "paste-html-to-govspeak"
-        } %>
+          <%= render "govuk_publishing_components/components/input", {
+            label: {
+              text: "Title"
+            },
+            value: @world_location_news.name,
+            name: "world_location_news[title]",
+            type: "text"
+          } %>
 
-        <%= location_form.check_box :active,
-                  label_text: "Active (can visitors click through from the world location list?)" %>
+          <%= render "components/govspeak-editor", {
+            label: {
+              text: "Mission statement",
+              heading_size: "l",
+            },
+            name: "world_location_news[mission_statement]",
+            id: "world_location_news_mission_statement",
+            value: @world_location_news.mission_statement,
+            rows: 10,
+            data_mod: {
+              module: "paste-html-to-govspeak"
+            }
+          } %>
 
```

Comment:
Annoyingly, when we add a checkbox via the publishing components checkbox component, we need to pass a false value "0" through to the controller so it sets the value to false.

If we don't do that you can uncheck the box and submit, but nothing is passed through to the controller in the params for the field. We can get round that like this.

```
<%= hidden_field_tag world_location_news[world_location_attributes][active]",  "0" %>
```

Reply:
Brill will add and include in the guide.